src/lib/datepicker/datepicker-intl.ts

```diff
@@ -6,7 +6,8 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Injectable, EventEmitter} from '@angular/core';
+import {Injectable} from '@angular/core';
+import { Subject } from 'rxjs/Subject';
```

Comment:
that's ok. however, it's better to apply a tool like prettier that format the code at pre-commit. That would be more efficient on code style management.